src/login/LoginFailure.jsx

```diff
@@ -64,7 +75,7 @@ const LoginFailureMessage = (props) => {
             check your spam folders or {supportLink}."
             values={{
               lineBreak: <br />,
-              email: <strong className="data-hj-suppress">{props.loginError.email}</strong>,
+              email: <strong className="data-hj-suppress">{context.email}</strong>,
```

Comment:
since context data is dynamic based on the error code, we should get it like `context?.email` to avoid any possible app crashes.
This comment holds true everywhere with `context`.